packages/app/src/elements/item-view.ts

```diff
@@ -508,11 +513,15 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         if (!this._nameInput.reportValidity()) {
             return;
         }
+        const oneMonthAgo = sub(new Date(), { months: 1 });
```

Comment:
Going back a month seems kind of arbitrary. Why not use the beginning of the Unix epoch (`new Date(0)`)?